migrating_from_ocp_3_to_4/planning-migration-3-4.adoc

```diff
@@ -69,38 +69,38 @@ In {product-title} 3.11, you upgraded your cluster by running Ansible playbooks.
 For more information, see xref:../updating/updating-cluster-between-minor.adoc#updating-cluster-between-minor[Updating clusters].
 
 [id="migration-considerations"]
-= Migration considerations
+== Migration considerations
```

Comment:
@apinnick Can you please check/peer review the changes to this file. This had a second level 1 heading in the file (which you can only have 1 in a file), so it's bumping everything below it properly.

Reply:
LGTM. I must have done that by accident when I made some changes recently. Thanks for catching it.